DEVELOPERS.md

```diff
@@ -327,27 +346,23 @@ if __name__ == "__main__":
     sys.exit(my_cli_plugin())
 ```
 
-For pixelator to find this you need to define an `entrypoint https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-for-plugins`_.
-Exactly how to do this depends on which packaging method you use to build your python package. In the link above you can find
-instructions for some common scenarios. The name you need use for your entry point is: `pixelator.cli_plugin`, so if you are using,
-`pyproject.toml` to configure your project it could look something like this:
+For pixelator to find this you need to define an [entrypoint](https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-for-plugins).
 
-```{code-block} toml
-:caption: pyproject.toml
+Exactly how to do this depends on which packaging method you use to build your python package. In this [link](https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-for-plugins) you can find
+instructions for some common scenarios. The name you need to use for your entry point is: `pixelator.cli_plugin`, so if you are using, `pyproject.toml` to configure your project it could look something like this:
```

Comment:
```suggestion
instructions for some common scenarios. The name you need to use for your entry point is: `pixelator.cli_plugin`, so if you are using `pyproject.toml` to configure your project, it could look something like this:
```